cmd/kube-proxy/app/server.go

```diff
@@ -476,14 +476,17 @@ with the apiserver API to configure the proxy.`,
 			cliflag.PrintFlags(cmd.Flags())
 
 			if err := initForOS(opts.WindowsService); err != nil {
-				klog.Fatalf("failed OS init: %v", err)
+				klog.ErrorS(err, "failed OS init")
+				os.Exit(255)
```

Comment:
os.Exit(1)  ?

Reply:
https://pkg.go.dev/k8s.io/klog@v0.3.2#Fatalf     it is os.Exit(255),  is it ok? Thank very much.

Reply:
/cc @ehashman

Reply:
Maybe this is helpful
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

Reply:
@yangjunmyfm192085  Thank you for your guidance, I have fixed it. Thanks very much again.